### ISSUE TITLE
[Global Styles] Disabled buttons' colors are not accessible

### ISSUE THREAD

**Issue by andycochran** [zurb/foundation-sites#7379]
_This is related to #7378, #7377, #7376, #7104, and #7103._

Accessibility Question: 
Does it matter that an element is accessible if it cannot be interacted with? 
(Perhaps @Owlbertz might weigh in.)

**Comment by Owlbertz** [comment]
I think there are two things to consider: people that are (partly) not able to see colors and screen readers (for people that cannot see at all). 
For the first case, the disabled style is not accessible for sure because of the pretty low (its 3.01 to 1, needs to be at least 4.5 to 1). I am not sure if there are any hard requirements for the style of disabled states, maybe the cursor could change to the disabled state when hovering? 
Regarding screen readers everything should be fine as long as there is the disabled attribute. This is currently not set, but I think this - rather than the class - should indicate the disabled state.

**Comment by zurbrandon** [comment]
Based on this conversation i'm inclined to say we could use a disabled cursor on them but that its ok they didn't have a correct contrast ratio. 

http://ux.stackexchange.com/questions/80881/do-disabled-buttons-still-need-to-be-contrast-compliant-for-accessibility

**Comment by andycochran** [comment]
I agree. 

Since W3C says… 

> 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
>
>*  (Level AA) Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
* Incidental: Text or images of text that are part of an **inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
* Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

I'm closing this.

**Comment by andycochran** [comment]
Of course, I might make a PR with 
```
.button[aria-disabled="true"] { 
  cursor: not-allowed; 
}
``` 
Is this what you're suggesting, @Owlbertz?

**Comment by gakimball** [comment]
@andycochran There's also `.button.disabled` and `button[disabled]`.

*Although*, I'm inclined to say we should recommend just one way to indicate a disabled button, and then only have CSS for that.

@Owlbertz @andycochran What's the difference between using `disabled` and `aria-disabled`?

**Comment by gakimball** [comment]
I'm collecting all of the contrast issues into one: #7387
